tm2/pkg/std/memfile.go

```diff
@@ -37,8 +37,16 @@ func (mempkg *MemPackage) IsEmpty() bool {
 	return len(mempkg.Files) == 0
 }
 
+const pathLengthLimit = 256
+
 var (
-	rePkgName      = regexp.MustCompile(`^[a-z][a-z0-9_]*$`)
+	rePkgName = regexp.MustCompile(`^[a-z][a-z0-9_]*$`)
+	/*
+		in case we use RE2 external lib (e.g. dlclark/regexp2), we can do this
+		^(?=.{0,256}$)gno\.land\/(?:p|r)(?:\/_?[a-z]+[a-z0-9_]*)+$
+		with lookahead operation to limit the length
+		but for now we just check it with const pathLengthLimit in bytes.
+	*/
```

Comment:
No, we will not use the package you mentioned; and we will not use any lookahead/behind/whatever syntaxes, either. I will not delve into the considerations of why re2 (and similar, like Go's standard `regexp`) are better for handling arbitrary data on the server-side, as [Russ Cox has already explained this in great detail](https://swtch.com/~rsc/regexp/regexp1.html). So, please, remove this comment.